examples/swr-site/styles.css

```
@@ -1,3 +1,4 @@
+@import 'nextra-theme-docs' layer(nextra-base);
```

```suggestion
@import 'nextra-theme-docs/style.css' layer(nextra-base);
```

let’s be more explicit, so `style` field in package.json can be removed too

also, can you give an example (in swr-site example) how cascade layers do css priority before/after?

---

`postcss-import` cannot recognize conditional exports in `package.json`, so I changed it to:

```css
@import 'nextra-theme-docs/dist/style.css';
```

---

Both `swr-site/styles.css` and `nextra-theme-docs/style.css` define styles for the body, with the same specificity of `0 0 1`.

- **Before using cascade layers**: The browser applies the styles of the latter selector based on the final compiled CSS order.
- **After using cascade layers**: Unlayered styles will be applied with higher priority, without considering the order of the styles.

**nextra-theme-docs**

```css
body, .nextra-nav-container-blur, .nextra-toc-footer, .nextra-sidebar-footer {
    background-color: rgb(var(--nextra-bg));
}
```

**swr-site**

```css
body {
    background: linear-gradient(to bottom, rgba(255, 255, 255, 0) 0%, rgb(var(--nextra-bg)) 300px), 
    fixed 0 0 / 20px 20px radial-gradient(#d1d1d1 1px, transparent 0), 
    fixed 10px 10px / 20px 20px radial-gradient(#d1d1d1 1px, transparent 0);
}
```

---

amazing improvement! It would be nice to mention this in docs as well